.github/workflows/deployAzureBackendDemo.yml

```diff
@@ -1,11 +1,9 @@
 name: Deploy Demo Backend Azure Resources
 
 on:
+  schedule:
+    - cron: '30 9 * * *' # Daily at 09:30 UTC
```

Comment:
I think this is 4:30 AM ET... Just confirming we want middle-of-the-night deploys. Is the demo going down a pager offense?

Reply:
It seemed better to have it _be_ down first thing in the morning than to have it _go_ down after people had started work on other things. But yes that's one reason I wanted comments. 🙂

I don't think that either of these environments warrants a nighttime page, but demo needs to be available and up to date during the day (generally midday from what I can tell), hence this suggestion.